doc/governance/gov_api_schemas/upgrading_from_classic.rst

```diff
@@ -0,0 +1,251 @@
+Upgrading From Classic API
```

Comment:
This is the bulk of the new content. Contains tables to point users from a classic endpoint to its replacement, with links to our existing (generated-from-OpenAPI) description of each.

Looks like this:
![image](https://github.com/microsoft/CCF/assets/6000239/8a45d436-6225-4bd1-8f76-a36be3996e62)